Sources/LiveViewNative/Property Wrappers/LiveContext.swift

```diff
@@ -133,6 +133,14 @@ public struct LiveContext<R: RootRegistry>: DynamicProperty {
     ) -> [NodeChildrenSequence.Element] {
         element.children().filter(Self.isTemplateElement(template))
     }
+    
+    func children(
+        of element: ElementNode
+    ) -> [NodeChildrenSequence.Element] {
+        element.children().filter {
+            !($0.asElement()?.attributes.contains(where: { $0.name.rawValue == "template" }) ?? false)
```

Comment:
A few probably dumb questions:

1. I noticed this function calls `asElement` on each child node before doing the conditional check as opposed to calling `asElement` only on child nodes that contain the `template` attribute; is this necessary and if not, do you think it incurs an additional performance cost that is worth refactoring?
2. Any reason for the ternary instead of using `guard`?

Reply:
Good call, I simplified the implementation.